YOUR_ADMIN/zca_bootstrap_colors.php

```diff
@@ -14,28 +14,40 @@
 
 $groupID = $db->Execute($sqlGroup);
 
-$_GET['gID'] = $groupID->fields['configuration_group_id'];
+$gID = $groupID->fields['configuration_group_id'];
+if (empty($gID)) {
+  $gID = 1;
```

Comment:
Nope, that's going to result in the "My Store" information being overwritten.  That configuration_group_id should be present, unless there was an issue with the template's installation.

Better to message the admin and return to the admin's main menu.